src/gui/widgets/ControllerView.cpp

```diff
@@ -255,6 +268,8 @@ void ControllerView::contextMenuEvent( QContextMenuEvent * )
 	collapse->setDisabled(gui->getControllerRackView()->allCollapsed());
 	QAction * expand = contextMenu->addAction( embed::getIconPixmap( "stepper-down" ), tr( "&Expand all controllers" ), this,SIGNAL( expandAll() ) );
 	expand->setDisabled(gui->getControllerRackView()->allExpanded());
+	contextMenu->addAction( embed::getIconPixmap( "cancel" ), tr( "&Move up" ), this, SLOT( moveUp() ) );
```

Comment:
Cross icon looks confusing for "Move up" and "Move down" actions. Use stepper-up and stepper-down or arp_up and arp_down instead?

Reply:
Sure. :+1